core/src/test/scala/com/rallyhealth/vapors/core/fql/PrinterSpec.scala

```diff
@@ -43,27 +43,25 @@ class PrinterSpec extends AnyWordSpec {
     "print an expression that selects a value using the NamedLens select" in {
       val q = {
         or(
-          __.withFactsOfType(FactTypes.ProbabilityToUse)
-            .withValuesAt(_.select(_.scores).atKey("weightloss"))
-            .whereAnyValue {
-              and(exists(__ > 0.5), exists(__ < 0.3))
+          withType(FactTypes.ProbabilityToUse)
+            .whereAnyFactValue { value =>
+              value.at(_.select(_.scores).atKey("weightloss")).exists { score =>
+                score > 0.5 and score < 0.3
+              }
             },
-          __.withFactsOfType(FactTypes.WeightMeasurement)
-            .whereAnyValue(__ > 150),
+          withType(FactTypes.WeightMeasurement)
+            .whereAnyFactValue(_ > 150),
         )
       }
       assertResult(FactsMatch(NonEmptyList.of(JoeSchmoe.weight))) {
         evalWithFacts(JoeSchmoe.facts)(q)
       }
       assertResult(
-        // TODO: This query looks wrong...
-        //       there is a mismatch between how value selection works within .exists() and conditional expressions
         "_.collect {" +
           " case f: Fact[Example.Probs] =>" +
-          " f.exists(_.value.scores['weightloss'] .exists(_where x > 0.5) and .exists(_where x < 0.3)) " +
+          " f.exists(_.value.scores['weightloss'] .exists(_ where x > 0.5 and  where x < 0.3)) " +
```

Comment:
The expression seems to have gone from `_where` in both cases previously to something inconsistent now?

Is this a bug in a pretty printer?

Reply:
This code is unused and I plan to remove it anyway.

Reply:
See https://github.com/jeffmay/vapors/pull/15